org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/context/BaseWorkerContext.java

```diff
@@ -1102,7 +1102,13 @@ public ValueSetExpansionOutcome expandVS(ValueSet vs, boolean cacheOk, boolean h
       return new ValueSetExpansionOutcome("This value set is not expanded correctly at this time (will be fixed in a future version)", TerminologyServiceErrorClass.VALUESET_UNSUPPORTED, false);
     }
     
-    Parameters p = pIn.copy();
+    Parameters p = getExpansionParameters(); // it's already a copy
+    if (p == null) {
```

Comment:
Is it even possible for p to be null here without a null exception being thrown in getExpansionParameters?

There should be a null check in getExpansionParameters (that's my mistake there).

Reply:
well, it can be null - found it in the tests. You can clean it up if you want